src/wayland/symbols.map

```diff
@@ -205,3 +205,21 @@ global:
   };
   local: *;
 };
+
+MIRWAYLAND_1.3 {
```

Comment:
We've not released 1.2 yet, so you can add to that.

Reply:
Addressed